### ISSUE TITLE
TypeError: Cannot read property 'expires' of undefined when using the crypto option

### ISSUE THREAD

**Issue by aa15032261** [jdesboeufs/connect-mongo#393]
This error only appears when crypto.secret is set. 

I have done some debugging and the issue seems to be that the session object is over stringified while the crypto option is used. Therefore, the session json is not converted to an object while the createSession function treats the session parameter as an object. 

At MongoStore.js line 197, the session is still a json string when it is passed to callback.:
```
const s = session && this.transformFunctions.unserialize(session.session);
if (this.options.touchAfter > 0 && (session === null || session === void 0 ? void 0 : session.lastModified)) {
    s.lastModified = session.lastModified;
}
this.emit('get', sid);
callback(null, s);
```

Trace: 
```
TypeError: Cannot read property 'expires' of undefined
    at MongoStore.Store.createSession (...\node_modules\express-session\session\store.js:87:29)
    at inflate (...\node_modules\express-session\index.js:368:13)
    at ...\node_modules\express-session\index.js:495:11
    at ...\node_modules\connect-mongo\build\main\lib\MongoStore.js:202:17
```

**Comment by mingchuno** [comment]
Yes, I am aware of this too when migrating to Typescript but I didn't have time to fix it yet. It could be if you can provide me a PR for the fix. 🙏🏻

**Comment by mingchuno** [comment]
Related to #372

**Comment by jas-** [comment]
As an FYI, a new release for kruptein should address this with `v3.0.0`. I have been testing with the example project and should have a PR soon

**Comment by jas-** [comment]
Landed in ##394, I have gotten this to pass in my own dev env. but am not able to determine where the error is from the github workflow

**Comment by mingchuno** [comment]
fixed in 4.2.2
